docs/enterprise/index.md

```diff
@@ -32,6 +32,9 @@ For more in depth information, here are some related articles:
 - [Blockchain Use Cases and Applications by Industry](https://media.consensys.net/enterprise-ethereum-blockchain-use-cases-and-applications-by-industry-3914d1210049)
 - [EY releases zero-knowledge proof blockchain transaction technology to the public domain to advance blockchain privacy standards](https://www.ey.com/en_gl/news/2019/04/ey-releases-zero-knowledge-proof-blockchain-transaction-technology-to-the-public-domain-to-advance-blockchain-privacy-standards)
 - [Introduction to Quorum: Blockchain for the Financial Sector](https://medium.com/blockchain-at-berkeley/introduction-to-quorum-blockchain-for-the-financial-sector-58813f84e88c)
+- [Official Quorum Documentation](http://docs.goquorum.com/en/latest)
```

Comment:
I don't think a link to documentation is fitting in this "Why Enterprise Ethereum" section. Can we instead put this below the "Quorum" link in the "Protocol and Infrastructure" section?

Reply:
updated, thanks